vmd/cv_dashboard/pkgIndex.tcl

```diff
@@ -1,4 +1,4 @@
-# Tcl package index file, version 1.1
+# Tcl package index file, version 1.2
```

Comment:
This is a trick: it's the version number of the Tcl utility that created the index file.

That said, I think it is time to bump the Dashboard version.